build.gradle.kts

```diff
@@ -53,6 +53,8 @@ buildscript {
 plugins {
     alias(libs.plugins.grazie)
     id("ai.kotlin.dokka")
+    id("org.jetbrains.kotlinx.kover") version "0.7.2"
```

Comment:
it is better to put it together with version into `libs.versions.toml` and then reference via `alias`